### DISCUSSION TITLE
What is the File ID for Ascalon City?

### DISCUSSION THREAD

**Discussion by Miniputs** [Jonathan-Greve/GuildWarsMapBrowser#14] [Q&A] [answered]
I have looked at the reddit screenshot but can't find the ID.
[https://preview.redd.it/guild-wars-map-browser-v5-0-exporting-to-blender-and-more-v0-58cqjytabp0c1.png?width=3840&format=png&auto=webp&s=1f12ea112797f86350d1c8b4a4da20b6916f07fc](url)

**Comment by Jonathan-Greve** [comment, marked as answer]
The File ID is `0x1B97D`. But for maps many of them have their names registered so you can search for them: 
![image](https://github.com/Jonathan-Greve/GuildWarsMapBrowser/assets/16852003/c64ba822-8378-4a5d-92e2-2f4fe7445758)

You can also rename files yourself using this window:
![image](https://github.com/Jonathan-Greve/GuildWarsMapBrowser/assets/16852003/a8696324-4a11-42e8-b5a8-fa1e1d028181)

which is useful for bookmarking files for yourself or sharing it with others.

**Reply by Miniputs** [reply]
There was a natural reason why I couldn't find it. I had not visited the area after I had reinstalled GW1. After revisit it showed up.

**Comment by NoobaSteve** [comment]
hello, I've downloaded and unzipped the zip master and the map browser exe and ran it, then i located the gw.dat but nothing shows up in the map browser even when i search for things, im not sure what i did wrong here.

**Reply by Jonathan-Greve** [reply]
> hello, I've downloaded and unzipped the zip master and the map browser exe and ran it, then i located the gw.dat but nothing shows up in the map browser even when i search for things, im not sure what i did wrong here.

The Guild Wars 1 dat right?
